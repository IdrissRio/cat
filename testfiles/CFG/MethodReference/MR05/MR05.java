public class MR2 {
  public void bar() {}
  public static void main(String[] args) {
    Thread t2 = new Thread(new MR2()::bar);
    t2.start();
  }
}