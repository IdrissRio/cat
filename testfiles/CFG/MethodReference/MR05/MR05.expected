NodeName: Entry[15] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: Entry[7] Successors: [Exit[8]]
NodeName: Exit[8] Successors: []
NodeName: Entry[29] Successors: [Exit[37]]
NodeName: Exit[37] Successors: []
NodeName: Entry[40] Successors: [ClassInstanceExpr[69]]
NodeName: ClassInstanceExpr[69] Successors: [ExprMethodReference[67]]
NodeName: ExprMethodReference[67] Successors: [ClassInstanceExpr[64]]
NodeName: ClassInstanceExpr[64] Successors: [VariableDeclarator[61]]
NodeName: VariableDeclarator[61] Successors: [VarAccess[80]]
NodeName: VarAccess[80] Successors: [MethodAccess[81]]
NodeName: MethodAccess[81] Successors: [Exit[54]]
NodeName: Exit[54] Successors: []

NodeName: Exit[16] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: Exit[8] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
NodeName: Exit[37] Predecessor: [Entry[29]]
NodeName: Entry[29] Predecessor: []
NodeName: Exit[54] Predecessor: [MethodAccess[81]]
NodeName: MethodAccess[81] Predecessor: [VarAccess[80]]
NodeName: VarAccess[80] Predecessor: [VariableDeclarator[61]]
NodeName: VariableDeclarator[61] Predecessor: [ClassInstanceExpr[64]]
NodeName: ClassInstanceExpr[64] Predecessor: [ExprMethodReference[67]]
NodeName: ExprMethodReference[67] Predecessor: [ClassInstanceExpr[69]]
NodeName: ClassInstanceExpr[69] Predecessor: [Entry[40]]
NodeName: Entry[40] Predecessor: []
